src/applications/vaos/components/ConfirmedAppointment.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Link } from 'react-router';
+import moment from 'moment';
+
+function formatDate(date) {
+  const parsedDate = moment(date, 'MM/DD/YYYY HH:mm:ss');
+
+  if (!parsedDate.isValid()) {
+    return '';
+  }
+
+  return parsedDate.format('MMMM D, YYYY');
+}
+
+function titleCase(str) {
```

Comment:
There's a titleCase function at `platform/utilities/data/titleCase`, we might be able to use that?

Reply:
Looks like platform/titleCase ignores intentional capitalization. So the name will not display properly if it is all caps.